.circleci/config.yml

```diff
@@ -106,14 +118,16 @@ commands:
         default: false
     steps:
```

Comment:
all the params for this job are unnecessary? remove the params

Reply:
Not sure what you're asking here. Don't the parameters need to be declared at the top of the config file?

Reply:
these params arent being used. unless i am wrong?